<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorProp</class>
 <widget class="QDialog" name="EditorProp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>440</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>290</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Editor / Debugger</string>
  </property>
  <property name="locale">
   <locale language="Spanish" country="Spain"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>13</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>0</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>0</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>190</red>
         <green>190</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>190</red>
         <green>190</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Editor</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>190</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>0</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>85</red>
         <green>0</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>190</red>
         <green>190</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>190</red>
         <green>190</green>
         <blue>190</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Compiler / Debugger</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="helpText">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>11</y>
     <width>291</width>
     <height>278</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>18</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Show Help</string>
   </property>
  </widget>
  <widget class="QLabel" name="fontLabel">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>55</y>
     <width>85</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Font Size</string>
   </property>
  </widget>
  <widget class="QLabel" name="tabLabel">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>83</y>
     <width>96</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tab Size</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>59</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Pixels</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="tabSize">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>79</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>20</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="tabSpaces">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>117</y>
     <width>152</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tabs as Spaces</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="showSpaces">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>140</y>
     <width>134</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Spaces</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>84</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Spaces</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="fontSize">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>53</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="driveCirc">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>390</y>
     <width>134</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Drive Circuit</string>
   </property>
  </widget>
  <widget class="QLabel" name="boardLabel">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>317</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Board</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ardBoard">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>310</y>
     <width>140</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string notr="true">Uno</string>
   </property>
   <item>
    <property name="text">
     <string>Uno</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mega</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nano</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Duemilanove</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Leonardo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Custom</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="compPath">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>238</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="avraInc">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>272</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="compPathLabel">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>238</y>
     <width>93</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Compiler Path</string>
   </property>
  </widget>
  <widget class="QLabel" name="avraIncLabel">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>275</y>
     <width>94</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Avra Inc Path</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="customBoard">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>349</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="customLabel">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>352</y>
     <width>94</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Custom Board</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>helpText</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>false</bool>
  </property>
  <property name="gridSnapY">
   <bool>false</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
